eng/targets/VisualStudio.targets

```diff
@@ -134,13 +134,33 @@
   </ItemGroup>
 
   <Target Name="_CheckDeploymentTargetVisualStudioVersion"
-		      Condition="'$(DeployExtension)' == 'true' and '$(CreateVsixContainer)' == 'true'"
+          Condition="'$(DeployExtension)' == 'true' and '$(CreateVsixContainer)' == 'true'"
           BeforeTargets="DeployVsixExtensionFiles"
-	       	DependsOnTargets="GetVsixDeploymentPath">
+          DependsOnTargets="GetVsixDeploymentPath">
 
     <Error Text="Deployment to Dev16 is not supported in this *-vs-deps branch. Use the corresponding non-VS branch." 
            Condition="$([MSBuild]::ValueOrDefault('$(ExtensionsPath)', '').Contains('16.0'))"/>
   </Target>
 
   <Import Project="GeneratePkgDef.targets" Condition="'$(MSBuildRuntimeType)' != 'Core' and '$(GeneratePkgDefFile)' == 'true' and '$(DesignTimeBuild)' != 'true' and '$(BuildingForLiveUnitTesting)' != 'true'" />
+
+  <!--
+    TODO: Workaround for a bug in Microsoft.VisualStudio.Extensibility.JsonGenerators.Sdk
+    Remove once we can update to a version that includes the fix.
```

Comment:
Is there a bug number tracking this?

Reply:
I have a PR in the Gladstone repo that will address this. Once we update to that version the build will fail, so there is no need for a tracking issue.